src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerExtensions.cs

```diff
@@ -54,6 +68,19 @@ public static IApplicationBuilder UseExceptionHandler(this IApplicationBuilder a
         /// <param name="configure"></param>
         /// <returns></returns>
         public static IApplicationBuilder UseExceptionHandler(this IApplicationBuilder app, Action<IApplicationBuilder> configure)
+        {
+            return app.UseExceptionHandler(configure, suppressExceptionLog: false);
+        }
+
+        /// <summary>
+        /// Adds a middleware to the pipeline that will catch exceptions, log them, and re-execute the request in an alternate pipeline.
+        /// The request will not be re-executed if the response has already started.
+        /// </summary>
+        /// <param name="app"></param>
+        /// <param name="configure"></param>
+        /// <param name="suppressExceptionLog"></param>
+        /// <returns></returns>
+        public static IApplicationBuilder UseExceptionHandler(this IApplicationBuilder app, Action<IApplicationBuilder> configure, bool suppressExceptionLog)
```

Comment:
I don't think we need several new overloads, this is very niche. The new property on ExceptionHandlerOptions should be adequate.